### ISSUE TITLE
RChain Blog Article: RSong and the Music Business

### ISSUE THREAD

**Issue by TrenchFloat** [rchain/bounties#1009]
### Benefit to RChain
Discover RAM's with exceptional writing to contribute to the Official RChain Blog
Cultivation of a better bounty system-to-marketing org relationship
Develop greater global opinion using RAM writers vs employed writers
*Tell stories that reach musicians and music-lovers

### Budget and Objective
**Marketing Organization Expectations:** Write a follow-up article to [RSong: The Future of Music](https://www.rchain.coop/blog/rsong-the-future-of-music/) explaining the details of RSong's answer to music business issues in layman's terms. NOTE: This was my idea, which I am passing along to the Marketing Org in hopes of approval. 
**Article Structure:** Brief intro of RSong with link to [previous article](https://www.rchain.coop/blog/rsong-the-future-of-music/); explain specific problems in music industry including interview quotes from Peter Harris, Rudy Harper, and/or Greg Meredith; explain what RSong does to answer those problems, again using quotes; conclusion
**Article size:** Awaiting Marketing Org expectations
**Links to resources:** [RSong: The Future of Music](https://www.rchain.coop/blog/rsong-the-future-of-music/), [RSong: The Future of Sound - RCon3 Presentation](https://www.youtube.com/watch?v=pZmHOijNCxc)
**Imagery:** Only use Royalty-Free imagery sourced from the list below unless you are the owner of the photograph is screencapture. Here are some great links to thousands of free images:
https://pixabay.com/
https://www.pexels.com/
https://www.canva.com/photos/free/
http://photos.polarfox.com/
https://skitterphoto.com/photos/
**Is there a number of keywords that should be used?** The more the merrier. RChain Keywords.xlsx

Please make the issue SMART: Specific, Measurable, Achievable, Realistic, Timely.

Estimated Budget of Task: **$[400 - 250 Article / 150 Inforaphics and Interview Time]**
Approved By: **TBD. Reached out to Derek Beres**
Estimated Timeline Required to Complete the Task: **[2 weeks]** to allow for interviews
How will we measure completion? **[Article is written, edited, reviewed, approved by RChain Marketing Org, and published]**

See CONTRIBUTING.md for details on budget and reward process.

### Legal
Task Submitter shall not submit Tasks that will involve RHOC being transacted in any manner that (i) jeopardizes RHOC’s status as a software access token or other relevant and applicable description of the RHOC as an “asset”—not a security— or (2) violates, in any manner, applicable U.S. Securities laws.

**Comment by dckc** [comment]
I gather @TrenchFloat no longer plans to pursue this.

Anyone who does can re-open it.
